### DISCUSSION TITLE
How to find out one plot line  or scatter line is hoverd?

### DISCUSSION THREAD

**Discussion by spirit88888888** [epezent/implot#327] [Q&A]
I want to show a tooltip when one line is hoverd, but I can not figure out how to check if one line is hoverd. I can only find a flag for legned hoverd. Is there any easy way to check if one line is hoverd?

**Comment by epezent** [comment]
No, ImPlot does not currently perform hover tests for plot items. It's one of our most requested features, so please know it is something I am thinking about and want to support eventually. See also #56
